src/client/datascience/telemetry/kernelTelemetry.ts

```diff
@@ -35,35 +34,7 @@ export function sendKernelListTelemetry(
         }
     });
     trackKernelResourceInformation(resource, counters);
-    sendKernelTelemetryEvent(resource, Telemetry.KernelCount, stopWatch.elapsedTime, counters);
-}
-
-export function sendNotebookControllerCreateTelemetry(
```

Comment:
Removed duplicate function, no different from the other, just an argument, hence just reused.